common/src/main/java/com/vexsoftware/votifier/support/forwarding/redis/RedisForwardingSink.java

```diff
@@ -0,0 +1,79 @@
+package com.vexsoftware.votifier.support.forwarding.redis;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import com.vexsoftware.votifier.model.Vote;
+import com.vexsoftware.votifier.support.forwarding.ForwardedVoteListener;
+import com.vexsoftware.votifier.support.forwarding.ForwardingVoteSink;
+import redis.clients.jedis.Jedis;
+import redis.clients.jedis.JedisPool;
+import redis.clients.jedis.JedisPoolConfig;
+import redis.clients.jedis.JedisPubSub;
+
+import java.time.Duration;
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * @author AkramL
+ */
+public class RedisForwardingSink extends JedisPubSub implements ForwardingVoteSink {
+
+    public RedisForwardingSink(RedisCredentials credentials,
+                               RedisPoolConfiguration poolConfiguration,
+                               ForwardedVoteListener listener) {
+
+        this.channel = credentials.getChannel();
+        this.listener = listener;
+
+        JedisPoolConfig jedisPoolConfig = new JedisPoolConfig();
+
+        jedisPoolConfig.setMaxTotal(poolConfiguration.getMaxTotal());
+        jedisPoolConfig.setMaxIdle(poolConfiguration.getMaxIdle());
+        jedisPoolConfig.setMinIdle(poolConfiguration.getMinIdle());
+        jedisPoolConfig.setMinEvictableIdleTime(Duration.ofMillis(poolConfiguration.getMinEvictableIdleTime()));
+        jedisPoolConfig.setTimeBetweenEvictionRuns(Duration.ofMillis(poolConfiguration.getTimeBetweenEvictionRuns()));
+        jedisPoolConfig.setBlockWhenExhausted(poolConfiguration.isBlockWhenExhausted());
+
+        this.pool = new JedisPool(jedisPoolConfig,
+                credentials.getHost(),
+                credentials.getPort(),
+                5000,
+                credentials.getPassword());
```

Comment:
Same as the comment I left on RedisForwardingVoteSource

Reply:
Will be resolved later this day